cogs/logging.py

```diff
@@ -11,6 +11,7 @@
 class Logging(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
+        self.logging_channel = env.get_logging_channel_name()
```

Comment:
I think we might want to get the channel object here right away, instead of doing that in every listener separately, so we can just send messages immediately.